docker-compose.yml

```
@@ -39,4 +39,4 @@ services:
     entrypoint: ["sh", "-c", "sleep infinity"]
     environment:
       - TZ=Asia/Tokyo
-      - LC_ALL=en_US.UTF-8
+      - LC_ALL=en_US.UTF-8
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)